documentation/tutorial.md

```diff
@@ -68,3 +71,622 @@ tutorial overlord    localhost/localhost  2.9.37   unsupported  23:20:53Z
+      lastHeartbeatRecv: ISODate("2022-12-02T14:39:51.882Z"),
+      pingMs: Long("0"),
+      lastHeartbeatMessage: '',
+      syncSourceHost: '10.23.62.156:27017',
+      syncSourceId: 0,
+      infoMessage: '',
+      configVersion: 5,
+      configTerm: 1
+    },
+    {
+      _id: 2,
+      name: '10.23.62.243:27017',
+      health: 1,
+      state: 2,
+      stateStr: 'SECONDARY',
+      uptime: 300,
+      optime: { ts: Timestamp({ t: 1669991990, i: 1 }), t: Long("1") },
+      optimeDurable: { ts: Timestamp({ t: 1669991990, i: 1 }), t: Long("1") },
+      optimeDate: ISODate("2022-12-02T14:39:50.000Z"),
+      optimeDurableDate: ISODate("2022-12-02T14:39:50.000Z"),
+      lastAppliedWallTime: ISODate("2022-12-02T14:39:50.020Z"),
+      lastDurableWallTime: ISODate("2022-12-02T14:39:50.020Z"),
+      lastHeartbeat: ISODate("2022-12-02T14:39:51.861Z"),
+      lastHeartbeatRecv: ISODate("2022-12-02T14:39:52.372Z"),
+      pingMs: Long("0"),
+      lastHeartbeatMessage: '',
+      syncSourceHost: '10.23.62.55:27017',
+      syncSourceId: 1,
+      infoMessage: '',
+      configVersion: 5,
+      configTerm: 1
+    }
+  ],
+  ok: 1,
+  '$clusterTime': {
+    clusterTime: Timestamp({ t: 1669991990, i: 1 }),
+    signature: {
+      hash: Binary(Buffer.from("dbe96e73cf659617bb88b6ad11152551c0dd9c8d", "hex"), 0),
+      keyId: Long("7172510554420936709")
+    }
+  },
+  operationTime: Timestamp({ t: 1669991990, i: 1 })
+}
+```
+
+Now exit the shell by typing:
+```
+exit
+```
+
+
+### Remove replicas:
+To remove a replica type:
+```
+juju remove-unit mongodb/2
+```
+
+You’ll know that the replica was successfully removed when `watch -c juju status --color` reports:
+```
+Every 2.0s: juju status --color                                                                                                                                                ip-172-31-11-104: Fri Dec  2 14:44:25 2022
+
+Model     Controller  Cloud/Region         Version  SLA          Timestamp
+tutorial  overlord    localhost/localhost  2.9.37   unsupported  14:44:25Z
+
+App      Version  Status  Scale  Charm    Channel   Rev  Exposed  Message
+mongodb           active      2  mongodb  dpe/edge   96  no       Replica set primary
+
+Unit        Workload  Agent  Machine  Public address  Ports      Message
+mongodb/0*  active    idle   0        10.23.62.156    27017/tcp  Replica set primary
+mongodb/1   active    idle   1        10.23.62.55     27017/tcp  Replica set secondary
+
+Machine  State    Address       Inst id        Series  AZ  Message
+0        started  10.23.62.156  juju-d35d30-0  focal       Running
+1        started  10.23.62.55   juju-d35d30-1  focal       Running
+
+```
+
+As previously mentioned you can trust that Charmed MongoDB removed this replica correctly. This can be checked by verifying that the new URI (where the removed host has been excluded) works properly.
+
+
+## Passwords:
+### Retrieving the admin password:
```

Comment:
It is generally not a good idea to have headings running on from each other. I would add a little text here explaining that all access to the DB requires a password and the following sections explain how to get and set